src/Components/CreateImageWizard/steps/Packages/PackageRecommendations.tsx

```diff
@@ -77,7 +77,12 @@ const PackageRecommendations = () => {
           },
         });
 
-        if (response && response.data && response.data.packages.length > 0) {
+        if (
```

Comment:
is that change related to the pr?

Reply:
This is not related. However, we access the `response.data.packages`. This was a bug, that probably was not tested, but with import, I found out that this was failing. And I think we should not really use methods that are not safe for empty arrays before checking `response.data.packages`.